README.md

```diff
@@ -367,6 +367,51 @@ Class | Method | HTTP request | Description
   - TIMECARDS_WRITE: POST, PUT, and DELETE endpoints related to employee timecards
 
 
+## Pagination of V1 Endpoints
+
+V1 Endpoints return pagination information via HTTP headers. In order to obtain
+response headers and extract the `batch_token` parameter you will need to follow
+the following steps:
+
+1. Use the full information endpoint methods of each API to get the response HTTP
+Headers. They are named as their simple counterpart with a `with_http_info` suffix.
+Hence `listEmployeeRoles` would be called `list_employee_roles_with_http_info`. This
+method returns an array with 3 parameters: `response`, `http_status`, and
+`http_headers`.
+2. Use `batch_token = api_client.get_v1_batch_token_from_headers(http_headers)`
+to extract the token and proceed to get the following page if a token is present.
+
+### Example
+
+```ruby
+# load the gem
+require 'square_connect'
+# setup authorization
+SquareConnect.configure do |config|
+  # Configure OAuth2 access token for authorization: oauth2
+  config.access_token = 'YOUR ACCESS TOKEN'
+end
+
+api_instance = SquareConnect::V1EmployeesApi.new
+api_client = api_instance.api_client
+
+opts = {
+  order: "order_example", # String | The order in which employees are listed in the response, based on their created_at field.Default value: ASC
```

Comment:
Instead of `order_example`, I'd suggest a valid value since this code is likely to be copy-pasted.